src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ILLink.targets

```diff
@@ -14,7 +14,7 @@ This file contains the .NET 5-specific targets to customize ILLink
       Runtime libraries feature switches defaults
       Available feature switches: https://github.com/dotnet/runtime/blob/master/docs/workflow/trimming/feature-switches.md
      -->
-    <DebuggerSupport Condition="'$(DebuggerSupport)' == '' and '$(Configuration)' != 'Debug'">false</DebuggerSupport>
+    <DebuggerSupport Condition="'$(DebuggerSupport)' == '' and !$(Configuration.Contains ('Debug'))">false</DebuggerSupport>
```

Comment:
We can probably use `AndroidUseDebugRuntime` here , see https://github.com/xamarin/xamarin-android/blob/d794534cebc3378326f6220dc54218616b9e5850/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L215 ?